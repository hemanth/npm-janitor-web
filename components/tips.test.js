import React from 'react'
import Tips from './tips'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  let module = {
    errors: ['name is not valid'],
    warnings: ['Missing recommended field: contributors'],
    suggestions: ['Missing recommended field: engines']
  }
  const tree = renderer.create(
    <Tips module={module}/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
