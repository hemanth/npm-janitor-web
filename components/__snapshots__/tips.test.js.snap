// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div>
       
      1
       
      error
    </div>
    <div>
      - 
      name is not valid
    </div>
    <br />
  </div>
  <div>
    Add recommended 
    field
    : 
    <span>
      <span
        className="lhecVm"
      >
        contributors
      </span>
      <br />
      <br />
    </span>
  </div>
  <div>
    Add optional 
    field
    : 
    <span>
      <span
        className="lhecVm"
      >
        engines
      </span>
      <br />
      <br />
    </span>
  </div>
</div>
`;
